tools/reviewer/job/sync/RecipientGitHubDiff.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+import com.google.startupos.tools.reviewer.localserver.service.Protos;
+import org.kohsuke.github.GHIssueComment;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHPullRequestReviewComment;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RecipientGitHubDiff {
+  private GHPullRequest pullRequest;
+  private List<GHPullRequestReviewComment> commentList;
+  private String repoName;
+
+  public RecipientGitHubDiff(String repoName, int diffNumber, String login, String password)
+      throws IOException {
+    this.repoName = repoName;
+    GitHub gitHub = GitHub.connectUsingPassword(login, password);
+    GHRepository repository = gitHub.getRepository(repoName);
+    pullRequest = repository.getPullRequest(diffNumber);
+    commentList = pullRequest.listReviewComments().asList();
+  }
+
+  public Protos.Diff getDiff() throws IOException {
+
+    Protos.Diff.Builder diff = Protos.Diff.newBuilder();
+
+    // diff.setId();
+    diff.setAuthor(Protos.Author.newBuilder().setEmail(pullRequest.getUser().getEmail()).build());
+
+    //    diff.addReviewer(Protos.Reviewer.newBuilder()
+    //        .setEmail(pullRequest.listReviews().asList().get(1).)
+    //        .setNeedsAttention()
+    //        .setApproved()
+    //        .build());
+    diff.setDescription(pullRequest.getBody());
+    //    diff.setBug();
```

Comment:
We won't sync bug, status and workspace initially, you can also add a note for that.

Reply:
Done.